src/pages/signin/LoginForm.js

```diff
@@ -94,6 +96,13 @@ function LoginForm(props) {
         [props.account, props.closeAccount, input, setFormError, setLogin],
     );
 
+    const setEnableMemoryOnlyKeys = () => {
+        if (!window) {
+            return;
```

Comment:
Should we add a log line here to confirm that we didn't go out through this early return?

Reply:
I _thought_ that `window` would not be defined on native platforms, but is seems like it is? So maybe this check isn't even required.

Reply:
I think JS is running on all platforms. is using enableMemoryOnlyKeys going to break native?

Reply:
It doesn't seem to break native from my testing